test-runner/package.json

```diff
@@ -2,5 +2,6 @@
     "description": "runs the tests in parallel for faster results",
     "dependencies": {
         "async": "~0.8.0"
-    }
+    },
+    "main": "parallel-test-runner.js"

```

Comment:
This makes it so you can just say `node .\test-runner` instead of having to specify which js file in this folder to run.